<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deedsBCV</class>
 <widget class="qMRMLWidget" name="deedsBCV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>847</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputParametersCollapsibleButton">
     <property name="toolTip">
      <string>Pick input volume sequence. Each time point will be registered to the fixed frame.</string>
     </property>
     <property name="text">
      <string>Inputs</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fixed volume: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="fixedVolumeSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>The moving volume will be transformed into this image space.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="baseName">
         <string/>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Moving volume: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="movingVolumeSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>This volume will be transformed into the fixed image space</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="baseName">
         <string/>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="pipelineCollapsibleButton">
     <property name="text">
      <string>Pipeline steps</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="rigidCheckBox">
        <property name="text">
         <string>Rigid</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="affineCheckBox">
        <property name="text">
         <string>Affine</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deformableCheckBox">
        <property name="text">
         <string>Deformable</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedParamsCollapsibleButton">
     <property name="text">
      <string>Advanced parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Number of levels:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="numLevelsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>42</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Initial grid spacing:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="gridSpacingSpinBox">
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Max search radius:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="searchRadiusSpinBox">
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Step quantisation:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="stepQuantisationSpinBox">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Regularisation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="regularisationSpinBox">
        <property name="value">
         <double>1.600000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputParametersCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Output volume: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Output transform: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="outputVolumeSelector">
        <property name="toolTip">
         <string>(optional) The moving image warped to the fixed image space. NOTE: You must set at least one output object (transform and/or output volume)</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="outputTransformSelector">
        <property name="toolTip">
         <string>(optional) Computed displacement field that transform nodes from moving volume space to fixed volume space. NOTE: You must set at least one output object (transform and/or output volume).</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLTransformNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 16px;
}</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="statusLabel">
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
     <property name="centerOnScroll">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>deedsBCV</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>fixedVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deedsBCV</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>movingVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deedsBCV</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputTransformSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deedsBCV</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
